.gitignore

```diff
@@ -45,6 +45,7 @@ junit-reports/
 # Terraform
 .terraform*
 *.tfstate
+*.tfstate.*
```

Comment:
Why not a single `*.tfstate*` line?

Reply:
Both are needed for files ending in `tfstate`